src/index.html

```diff
@@ -5,8 +5,309 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav>
+        <ul class="navigation">
+          <a href="#apple" class="navigation__links is-active">
+            Apple
+          </a>
+          <a href="#samsung" class="navigation__links">
+            Samsung
+          </a>
+          <a href="#smartphones" class="navigation__links">
+            Smartphones
+          </a>
```

Comment:
don`t forget to add empty line between sibling tags
![2023-02-13_16h43_59](https://user-images.githubusercontent.com/61904995/218488793-ad88f3a2-ec79-4953-a814-cb95f56a28ae.png)